internal/command/scale/count_machines.go

```diff
@@ -61,8 +62,12 @@ func runMachinesScaleCount(ctx context.Context, appName string, appConfig *appco
 		return err
 	}
 
+	defaultGuest := new(api.MachineGuest)
+	defaultGuest.SetSize(deploy.DefaultVMSize)
 	defaults := newDefaults(appConfig, latestCompleteRelease, machines, volumes,
-		flag.GetString(ctx, "from-snapshot"), flag.GetBool(ctx, "with-new-volumes"))
+		flag.GetString(ctx, "from-snapshot"), flag.GetBool(ctx, "with-new-volumes"), defaultGuest)
+
+	mach.ApplyFlagsToGuest(ctx, defaultGuest)
```

Comment:
this works but relies on a side effect, it would be more correct if placed after the `.SetSize()` line above, right before handing over _defaultGuest_  to `newDefaults()`